src/pages/Login.jsx

```
@@ -1,29 +1,77 @@
+      const user = response.data.user;
+
+      localStorage.setItem('token', token);
+      localStorage.setItem('userId', user.id);
+      localStorage.setItem('avatar', user.avatar);
+      localStorage.setItem('phone', user.phone);
+      setEmail('');
+      setPassword('');
+      dispatch(loadCartFromLocalStorage());
+      navigate('/');
+    } catch (e) {
+      if (e.response && e.response.data.errors) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)